src/hammer-vlsi/cli_driver_test.py

```diff
@@ -39,6 +42,199 @@ def generate_dummy_config(self, syn_rundir: str, config_path: str, top_module: s
 
         return config
 
+    def run_syn_to_par_with_output(self, config_path: str, syn_rundir: str,
+                                   par_rundir: str,
+                                   syn_out_path: str,
+                                   syn_to_par_out_path: str) -> None:
+        # Check that running the CLIDriver executes successfully (code 0).
+        with self.assertRaises(SystemExit) as cm:  # type: ignore
+            CLIDriver().main(args=[
+                "syn",  # action
+                "-p", config_path,
+                "--output", syn_out_path,
+                "--syn_rundir", syn_rundir,
+                "--par_rundir", par_rundir
```

Comment:
Is there a reason to give it this extra path? Are you checking that it doesn't use it?

Reply:
I think in this specific case there should be no use of par_rundir but I generally leave it in there for safety purposes. Do you want me to remove it?

Reply:
Hmm. I don't think its a big deal, I'll leave it up to you